apps/ssr-tests/package.json

```diff
@@ -21,10 +21,8 @@
     "@fluentui/public-docsite-resources": "^8.0.0-beta.5",
     "just-scripts-utils": "^1.1.1",
     "mocha": "^3.3.0",
-    "raw-loader": "^4.0.2",
     "react": "16.8.6",
-    "react-dom": "16.8.6",
-    "webpack": "^5.21.2"
```

Comment:
Should we really perform any changes in `ssr-tests`? IMO that should stay with Webpack 5.

Reply:
Ignore: I see, it will use a default one from the root.